platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -83,18 +85,56 @@ class ViewerRetrieveStudyData extends Component {
     });
   }
 
+  _handleSeriesLoadResult(error, studyMetadata, series) {
+    if (this.abortSeriesLoad) return;
+    const seriesLoadStats = this.seriesLoadStats[
```

Comment:
I'm wary of reusing the same name for `local` variables and object properties. It's akin to "variable shadowing", and can lead to confusing code. Can we pick out better names here?

Reply:
yup! good point.

Reply:
fixed!